packages/salesforcedx-vscode-core/test/vscode-integration/orgPicker/orgList.test.ts

```diff
@@ -169,45 +180,63 @@ describe('orgList Tests', () => {
         expect(authList[0]).to.equal('test-scratchorg1@example.com');
       });
 
-      it('should filter the list to only show scratch orgs associated with current default dev hub with an alias', async () => {
+      it('should filter the list to only show scratch orgs associated with current default dev hub', async () => {
         const authInfoObjects: OrgAuthorization[] = [
           dummyScratchOrgAuth1,
           dummyScratchOrgAuth2
         ];
-        const getAuthFieldsForStub = sandbox.stub(orgList, 'getAuthFieldsFor');
         getAuthFieldsForStub.withArgs(dummyScratchOrgAuth1.username).returns({
           devHubUsername: dummyDevHubUsername1
         });
         getAuthFieldsForStub
           .withArgs(dummyScratchOrgAuth2.username)
           .returns({ devHubUsername: dummyDevHubUsername2 });
-        defaultDevHubStub.returns('dev hub alias');
+        getDevHubUsernameStub.returns(dummyDevHubUsername1);
         getUsernameStub.resolves(dummyDevHubUsername1);
         getAllStub.returns([]);
 
         const authList = await orgList.filterAuthInfo(authInfoObjects);
         expect(authList[0]).to.equal(dummyScratchOrgAuth1.username);
       });
 
+      it('should filter the list to remove org authorizations that have errors', async () => {
+        const authInfoObjectsWithOneError: OrgAuthorization[] = [
+          dummyScratchOrgAuth1,
+          dummyScratchOrgAuthWithError
+        ];
+        getAuthFieldsForStub
+          .withArgs(authInfoObjectsWithOneError[0].username)
+          .returns({
+            devHubUsername: dummyDevHubUsername1
+          });
+        getDevHubUsernameStub.resolves(dummyDevHubUsername1);
```

Comment:
nit: it'd be nice to move this line below 215 so the two `getAuthFieldsForStub` withArgs are grouped together.

Reply:
done!